### PR TITLE
Add Aries Robotics as a sponsor

### PR DESCRIPTION
**Pull request by sedders123**

### CODE REVIEW

**Review by antoinepetty — APPROVED** [review]

**Review by raccube — APPROVED** [review]

File: _posts/2023-02-23-aries-robotics-sponsorship.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "Our New Sponsor: Aries Robotics"
+---
+
+<img src="{{ '/images/sponsors/aries-robotics.svg' | prepend: site.baseurl }}" />
```
Comment (PeterJCLaw) [inline _posts/2023-02-23-aries-robotics-sponsorship.md:5]: The logo feels a bit on the big side -- I'd be tempted to add some padding around it?

Comment (PeterJCLaw) [inline _posts/2023-02-23-aries-robotics-sponsorship.md:5]: Please could we have an `alt` tag on this image.

File: _config.yml [hunk 2]
```diff
@@ -111,6 +111,11 @@ sponsors:
     image: /images/sponsors/bittele.svg
     logo_width: default
 
+  - name: Aries Robotics
```
Comment (PeterJCLaw) [inline _config.yml:114]: What defines the ordering of the sponsors on the homepage? Would it make sense for Aries to be first as they're the biggest now?

File: _posts/2023-02-23-aries-robotics-sponsorship.md (hunk 1 above)
Comment (sedders123) [reply _posts/2023-02-23-aries-robotics-sponsorship.md]: [4fe4804](https://github.com/srobo/website/pull/490/commits/4fe4804e844478cb4b544e58cfae1f6de9437d9f)

Comment (sedders123) [reply _posts/2023-02-23-aries-robotics-sponsorship.md]: [4fe4804](https://github.com/srobo/website/pull/490/commits/4fe4804e844478cb4b544e58cfae1f6de9437d9f)

**Pull request merged**